test/src/test/java/jenkins/util/SetContextClassLoaderTest.java

```diff
@@ -20,7 +20,7 @@ void positive() throws Throwable {
     }
 
     private static void _positive(JenkinsRule r) throws ClassNotFoundException {
-        try (SetContextClassLoader sccl = new SetContextClassLoader(RealJenkinsRule.Endpoint.class)) {
+        try (SetContextClassLoader sccl = new SetContextClassLoader(RealJenkinsFixture.Endpoint.class)) {
```

Comment:
Amending #6575. I am not really sure why it was using this as an example to begin with, and not say `PluginManager.uberClassLoader`.

Reply:
I can follow-up with a PR correcting this if you want. Would however wait for https://github.com/jenkinsci/jenkins-test-harness/pull/1135 to me accepted / released.

Reply:
Addressed in https://github.com/jenkinsci/jenkins/pull/26020